code/Common/DefaultIOStream.cpp

```diff
@@ -63,7 +63,7 @@ inline int select_fseek(FILE *file, int64_t offset, int origin) {
 
 
 
-#if defined _WIN32 && (!defined __GNUC__ || __MSVCRT_VERSION__ >= 0x0601)
+#if defined _WIN64 && (!defined __GNUC__ || __MSVCRT_VERSION__ >= 0x0601)
```

Comment:
What about win32?

Reply:
In Msys2 clang32 environment this resolves to true, resulting in a build failure. `__MSVCRT_VERSION__` is `0xE00` in clang32.

Reply:
This change breaks x86-windows builds...